src/components/JobsPanelParameters/JobsPanelParameters.js

```diff
@@ -218,6 +225,7 @@ const JobsPanelParameters = ({
         }
 
         param.data.value = parametersState.selectedParameter.data.value
+        param.data.valueType = parametersState.selectedParameter.data.valueType
```

Comment:
Bug fix: the param type was not updated properly when editing an existing parameter.